backend/app/testbot/data_loader.py

```
@@ -1,13 +1,162 @@
+        # Extract structured data from description
+        clean_product["features"] = extract_product_features(original_description)
+        size_chart = parse_size_chart(original_description)
+        if size_chart:
+            clean_product["size_chart"] = size_chart
+    
+    # Handle price fields
+    if "discountedPrice" in clean_product:
+        try:
+            clean_product["discountedPrice"] = float(clean_product["discountedPrice"])
+        except (ValueError, TypeError):
+            pass
```

Ruff (0.8.2)

Use `contextlib.suppress(ValueError, TypeError)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(ValueError, TypeError)`

(SIM105)